### PR TITLE
Add support for Color attributes

### PR DESCRIPTION
**Pull request by Gary-Lockyer**
- Should the PyColor be in it's own file?
- Not convinced about the naming of is_list

### CODE REVIEW

File: omf-python/src/element.rs [hunk 1]
```diff
@@ -31,4 +32,51 @@ impl PyElement {
...
         PyGeometry(self.0.geometry.clone())
     }
+
+    #[getter]
+    fn color(&self) -> Option<PyColor> {
+        self.0.color.map(PyColor)
+    }
+}
+
+#[pyclass(name = "Color")]
```
Comment (jsutton24) [inline omf-python/src/element.rs:42]: fyi you can get the `__eq__()` method for free with:

```rust
#[derive(PartialEq, Eq)]
#[pyclass(name = "Color", eq)]
```

**Review by jsutton24 — APPROVED** [review]

**Review by antbrown — APPROVED** [review]

File: omf-python/src/element.rs [hunk 2]
```diff
@@ -31,4 +32,51 @@ impl PyElement {
...
+    #[getter]
+    fn blue(&self) -> u8 {
+        self.0[Self::BLUE]
+    }
+    #[getter]
+    fn alpha(&self) -> u8 {
+        self.0[Self::ALPHA]
+    }
+
+    fn as_list(&self) -> [u8; 4] {
```
Comment (andrew-catalyst) [inline omf-python/src/element.rs:79]: I wonder if Colors should just be exposed to python as an array of 4 u8 (similar to what we did with vertices) so no PyColor class.

Comment (Gary-Lockyer) [reply omf-python/src/element.rs:79]: I did this mainly to document the positions, i.e. reg, green, blue, alpha.

Comment (andrew-catalyst) [reply omf-python/src/element.rs:79]: That's the normal convention though. If you're going to pass it to something in python to do with color manipulation it'll need it to be an array anyway.

**Pull request merged**